README.md

```diff
@@ -85,6 +89,10 @@ Activate the virtualenv:
   $ source name of virtualenv/bin/activate
 
 ```
+Create a .env file add the following
+
+export FLASK_RYN="run.py"
+export SECRET = 'mysecretisgood'
```

Comment:
You can add your secret key to your .env so that you don't have to push it to github. Then SECRET will be = env.get[SECRET] or a similar kind of implementation.